Makefile

```diff
@@ -79,5 +81,31 @@ opensbi_jump.bin: opensbi
 	cp opensbi/build/platform/generic/firmware/fw_jump.bin opensbi_jump.bin
 	cp opensbi/build/platform/generic/firmware/fw_jump.elf opensbi_jump.elf
 
+
+u-boot:
+	git clone https://github.com/u-boot/u-boot.git
+	cd u-boot && git apply ../u-boot_patch.patch
+	cd u-boot && make CROSS_COMPILE=riscv64-linux-gnu- qemu-riscv64_smode_defconfig
+	cd u-boot && make CROSS_COMPILE=riscv64-linux-gnu-
+	cp u-boot/u-boot.bin u-boot.bin
+	rm -rf u-boot
+
+u-boot-ci-cd:
+	git clone https://github.com/u-boot/u-boot.git
```

Comment:
Same remark here, we need to fix a tag (you can check how we did it for OpenSBI).

While we are at it, we can also pass `--depth 1` to speed up the CI and reduce the bandwidth usage :)